python-avd/pyavd/j2tests/defined.py

```diff
@@ -59,11 +59,13 @@ def defined(
     -------
     bool
         True if variable matches criteria, False in other cases.
+
+    TODO: Remove the run_tests flag and internal logic only used for unit tests.
```

Comment:
ugh

Reply:
yeah old tech dept. Someone thought it was a good way of testing it.